repos/system_upgrade/el7toel8/actors/sssdfacts/tests/unit_test.py

```diff
@@ -9,7 +10,13 @@
 
 def get_config(content):
     config = configparser.ConfigParser()
-    config.readfp(StringIO(textwrap.dedent(content)))
+    if hasattr(config, 'read_file'):
+        config.read_file(StringIO(textwrap.dedent(content)))
+    else:
+        assert PY2
```

Comment:
again, do not use assert out of test functions.

Reply:
Ok, fixing...